frontend/src/metabase/core/components/AccordionList/AccordionListCell.styled.tsx

```diff
@@ -21,11 +21,6 @@ export const Content = styled.div<{ isClickable: boolean }>`
   padding: 0.5rem;
   cursor: ${props => (props.isClickable ? "pointer" : "default")};
   min-width: 0;
-
-  [data-element-id="list-item"]-content {
```

Comment:
it's not used anymore, [confirmed](https://metaboat.slack.com/archives/C0645JP1W81/p1720097398102519)